Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -72,25 +74,31 @@
   - type: Clothing
     sprite: Clothing/Belt/ce.rsi
   - type: Storage
-    # TODO: Fill this out more.
-    # whitelist:
-    #   tags:
-    #     - Wirecutter
-    #     - Crowbar
-    #     - CrowbarRed
-    #     - Screwdriver
-    #     - Flashlight
-    #     - Welder
-    #     - Wrench
-    #     - Painter
-    #     - GeigerCounter
-    #     - Flare
-    #     - Coilsofwire
-    #     -
-    #     -
-    #   components:
-    #     - SignalLinker
     capacity: 100
+    # TODO: Fill this out more.
```

Comment:
is this a TODO for this PR or for future PRs

Reply:
Future PRs